enterprise/glassfish.common/src/org/netbeans/modules/glassfish/common/StartTask.java

```diff
@@ -691,6 +691,9 @@ private StartupArgs createProcessDescriptor() throws ProcessCreationException {
         // append other options from startup extenders, e.g. for profiling
         appendStartupExtenderParams(optList);
 
+        optList.add("--add-opens java.base/java.lang=ALL-UNNAMED");
```

Comment:
I think this should be in netbeans.conf. See [PR-1581](https://github.com/apache/netbeans/pull/1581/files)

Reply:
@nicolaken the question is: are the opens directives targeted at the glassfish process or at the netbeans process

@pepness I would not be too surprised, if an application server uses reflection to change the JDK internals, so it is plausible from my perspective, that the module openes are required for the glassfish to work correctly.

Reply:
This is not needed to run the Netbeans IDE, it's only needed to launch Glassfish.
I have not tested, but because it spawn a new process, it should not inherit the Netbeans IDE arguments, and thus not work.